client/elife-theme/src/index.js

```diff
@@ -6,6 +6,7 @@ const gridUnit = 6
 
 export default {
   /* Colors */
+  colorBackground: '#fff',
```

Comment:
Why did you add this? You don't use it in here (`background-color` is set to `colorPrimary`). We removed it because it was not on `VARIABLES.md` but it is now on the wiki. We don't have it in invision.

Reply:
It's used in PubSweet components so if we don't have it those components don't render properly. Yet another reason why `VARIABLES.md` should be deleted.

Reply:
I just checked and yes, it's being currently used. Makes sense to add it.